### ISSUE TITLE
HTML minify step still sends JS minify-excluded scripts to JS minifier when bundler has inlined them.

### ISSUE THREAD

**Issue by usergenic** [Polymer/polymer-cli#958]
Root cause of why expected excluded JS is still getting minified/compiled:  https://github.com/Polymer/polymer-cli/issues/956#issuecomment-363232433

This is an unexpected behavior that bundling should essentially do an end-run around the exclusions specified.  Fixing this may require some bundler work to inject identifiers into inlined code so we have a way to apply an excluded filename pattern match.

**Comment by iSuslov** [comment]
This is very important to fix. I can not get rid of minifying `firebase-database.js` which breaks my app.

**Comment by usergenic** [comment]
@iSuslov did you try what @christophe-g tried in his comment as a fix? https://github.com/Polymer/polymer-cli/issues/956#issuecomment-363680112

**Comment by iSuslov** [comment]
@usergenic yes, nothing works due to this https://github.com/Polymer/polymer-cli/issues/1008

**Comment by iSuslov** [comment]
Forgot to add, I downgraded to 1.6.x and it works.
